CMakeLists.txt

```diff
@@ -161,6 +161,10 @@ if(ENABLE_PYTHON_BINDINGS)
     set(Boost_Python_lib "Boost::python${PYTHON_VERSION_MAJOR}")
     set(_Boost_PYTHON${PYTHON_VERSION_MAJOR}_HEADERS "boost/python.hpp")
     find_package(Boost COMPONENTS python${PYTHON_VERSION_MAJOR})
+  elseif(APPLE AND ${PYTHON_VERSION_MAJOR} EQUAL 2 AND ${PYTHON_VERSION_MINOR} EQUAL 7)
```

Comment:
why was this needed for the refactor?

Reply:
omg @dgearhart good catch!
this is a hot fix until https://github.com/valhalla/valhalla/pull/1342 is merged. not sure what to do here. I guess I will remove and push as it is at the moment broken anyways and will not break it more.